src/core/CMakeLists.txt

```diff
@@ -246,11 +246,15 @@ target_sources(openthread-radio PRIVATE
 if(OT_BUILTIN_MBEDTLS)
     target_link_libraries(openthread-ftd
         PUBLIC
```

Comment:
It seems `mbedtls` names are not exposed by the public headers of target `openthread-ftd`, so this may should be `PRIVATE`? Other targets depending on `mbedtls` may explicitly link it by themselves.

Reply:
We can move mbed libs private here. CLI and NCP target will also need to explicitly list mbed libs as a result.

Reply:
Yes, I think this is the way to do it. Ideally, they should just need to do:
```cmake
target_link_libraries(xxx-cli PRIVATE mbedtls ... )
```